slik_wrangler/preprocessing.py

```diff
@@ -436,7 +436,7 @@ def drop_duplicate(dataframe=None,columns=None,method='rows',display_inline=True
     elif method == 'columns':
         if columns is None:
             raise ValueError("columns: A list/string is expected as part of the inputs to columns, got 'None'")
-        dataframe = dataframe.drop_duplicates(subset=columns)
+        dataframe = dataframe.T.drop_duplicates().T
```

Comment:
What does .T do here?

Reply:
It translate the dataset, so the columns takes the position of the rows and the duplicate columns are then dropped, and finally it is translated to its original state.

Reply:
Yeah, it's the transpose function. Wondering why you had to do it here? Did you test it?